src/php/binaries.rs

```diff
@@ -26,6 +27,54 @@ pub(crate) fn current() -> String {
     "php".to_string()
 }
```

Comment:
The `current()` function is now unused, isn't it? Maybe it could be removed :)

Reply:
Yes, deleted